### ISSUE TITLE
campaign didn't compute

### ISSUE THREAD

**Issue by pierrealixt** [hotosm/MapCampaigner#586]
# Problem

This campaign.json didn't compute. Lambda functions don't show any errors.
```
{"campaign_creator": "nsmith", "campaign_managers": ["nsmith"], "campaign_status": "Start", "coverage": {}, "dashboard_settings": "", "description": "Collecting Schools across Jakarta", "edited_at": "2018-10-02", "edited_by": "nsmith", "end_date": "2018-12-31", "link_to_omk": false, "map_type": "", "name": "Jakarta Schools", "participants_count_per_type": {}, "remote_projects": [], "selected_functions": {"function-1": {"function": "FeatureAttributeCompleteness", "feature": "amenity=school", "attributes": {"amenity": ["school"], "name": []}, "type": "schools"}, "function-2": {"function": "CountFeature", "feature": "amenity=school", "attributes": {"amenity": ["school"], "name": []}, "type": "schools"}, "function-3": {"function": "MapperEngagement", "feature": "amenity=school", "attributes": {"amenity": ["school"], "name": []}, "type": "schools"}}, "start_date": "2018-01-01", "thumbnail": "https://s3-us-west-2.amazonaws.com/hotosm-fieldcampaigner-data-production/campaigns/2817d40cd67844249e166f183b38ea60/thumbnail.png", "total_participants_count": 0, "types": {"type-1": {"type": "schools", "feature": "amenity=school", "tags": {"amenity: school": [], "name": []}}}, "uploader": "nsmith", "uuid": "2817d40cd67844249e166f183b38ea60", "version": 4}
```
# Solution
Why?

**Comment by ccamara** [comment]
My campaign is also computing after several days of creation: https://campaigns.hotosm.org/campaign/2a07007f604742fab81b8111ffd54e12

**Comment by jessbeutler** [comment]
I am also having the same issue with a few campaigns:
https://campaigns.hotosm.org/campaign/f74df638bca34fffbdf4f923373f303d
https://campaigns.hotosm.org/campaign/dd7b3627dc6942dc9798ea094f8c680b

**Comment by smit1678** [comment]
Quick update here: it looks like reason for failed campaigns from @jessbeutler and @ccamara were the geojsons. The geojson features were `MultiPolygon` and these were causing errors. Changing the feature type to `Polygon` fixes the issue.

**Comment by pierrealixt** [comment]
@smit1678 do you still have those MultiPolygon geojsons?

**Comment by smit1678** [comment]
Yes!

https://gist.github.com/smit1678/da41ebb89d13432b087da99fbe27eb22
https://gist.github.com/smit1678/34a14c973eb1274ffb47559a13c4ffd3

**Comment by pierrealixt** [comment]
and to fix those campaigns, you edited the geometry type from "MultiPolygon" to "Polygon" in the geojson?

**Comment by smit1678** [comment]
Yes, I just directly edited the geojson to change the type to `Polygon` and then removed the array the polygon was in.

**Comment by pierrealixt** [comment]
@smit1678 what about this?
![hot-multipolygon](https://user-images.githubusercontent.com/35932320/48202709-52ceea80-e36e-11e8-8fbc-324eac6c3b19.gif)

**Comment by Cybereric** [comment]
I think I will try this 
I've created a [campaign](https://campaigns.hotosm.org/campaign/836d41cc4d5b464487c075f81e0dac5f) 15 days ago and it's still computing !!!

**Comment by smit1678** [comment]
@Cybereric it looks like the geojson you used in that campaign is linestrings and not polygons. Can you change your file to polygons and then create a new campaign to test? I'll delete the other campaign that didn't process.

**Comment by smit1678** [comment]
@pierrealixt Yeah I think a warning looks and works well. Will this error come up with linestrings as well?

**Comment by Cybereric** [comment]
I tried a first time to edit the geojson file by doing a "search / replace" for "linestring" to "polygon"
It did not work.
So I loaded the file in http://umap.openstreetmap.fr and there I could edit the sectors and transform them into polygons.
I downloaded the file to reload it in Map Campaigner.
The campaign is now functional >>> https://campaigns.hotosm.org/campaign/3b49da3b5ba64e899215bbdb0938333d
